.github/workflows/close-jira-issue.yml

```
@@ -0,0 +1,31 @@
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const m = context.payload.issue.title.match(/[A-Z]+-\d+/);
+            return m ? m[0] : '';
+
+      - name: Close Jira issue
+        if: steps.extract-key.outputs.result != ''
+        uses: atlassian/gajira-transition@v3
+        with:
+          issue: ${{ steps.extract-key.outputs.result }}
+          transition: 완료
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)